client/src/App.css

```diff
@@ -68,7 +68,7 @@
     cursor: pointer;  
 }
 
-.top-nav h4 {
+.top-nav h5 {
```

Comment:
There's `.top-nav h1` followed by `.top-nav h5`. Headers should form some sort of hierarchy such as:

- h1
  - h2
    - h3
  - h2

If you use h5 because of its default styling, you're doing it wrong. Use correct HTML tag and style it to your needs.